install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -0,0 +1,55 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_rhv.adoc
+////
+
+[id='configuring-rhv-objects_{context}']
+= Configuring Red Hat Virtualization objects
+
+To integrate {product-title} with Red Hat Virtualization, take the following
+actions as part of your
+xref:../install/host_preparation.adoc#install-config-install-host-preparation[host preparation].
+
+. To provide high availability in case of the loss of one hypervisor host,
+add each class of instance to a negative affinity group. See
+link:https://ovirt.org/develop/release-management/features/sla/vm-affinity/[VM Affinity].
+
+. To ensure that the {product-title} environment meets the
+xref:../install/prerequisites.adoc#hardware[minimum hardware requirements],
+create templates for virtual machines that use the following resources:
++
+[cols="1,7"]
+|===
+
+|Master nodes
+a|* Minimum 2 CPU Cores
+* 16 GB Memory
+* Minimum 10 GB root disk
+* Minimum 15 GB container image registry disk
+* 30 GB local volume disk
+* Minimum 25 GB etcd disk
+
+|Infrastructure nodes
+a|* Minimum 2 CPU Cores
+* 16 GB Memory
+* Minimum 10 GB root disk
+* Minimum 15 GB container image registry disk
+* 30 GB local volume disk
+* Minimum 25 GB Gluster registry disk
+
+|Application nodes
+a|* 2 CPU Cores
+* 8 GB Memory
+* Minimum 10 GB root disk
+* Minimum 15 GB container image registry disk
+* 30 GB local volume disk
+
+|Load balancer node
+a|* 1 CPU Core
+* 4 GB Memory
+* 10 GB root disk
+
+|===
+
+. Create DNS entries for the routers and for the {product-title} web console.
```

Comment:
Is this enough information?

Reply:
I think that should have some more content. 

Create DNS entries for the routers and for the {product-title} web console. For the router, it is suggested to provide entries for all infrastructure instances and use DNS round robin to allow for the router to pass traffic to applications.

For the {product-title} web console an entry should be placed into DNS using the IP address of the Load Balancer node.